app/jobs/identities_backfill_job.rb

```diff
@@ -0,0 +1,41 @@
+class IdentitiesBackfillJob
+  # This is a short-term solution to backfill data requiring a table scan.
+  # This job can be deleted once it's done.
+
+  queue_as :low
+
+  # Let's give us the option to fine-tune this on the fly
+  BATCH_SIZE_KEY = 'IdentitiesBackfillJob.batch_size'.freeze
+  CACHE_KEY = 'IdentitiesBackfillJob.position'.freeze
+
+  def perform
+    position = Rails.cache.read(CACHE_KEY).to_i || 0
+    batch_size = Rails.cache.read(BATCH_SIZE_KEY).to_i || 500_000
+
+    # max id today is in the 184M range
+    return true if position >= 185_000_000
+
+    sp_query = <<~SQL
+UPDATE identities
+SET last_consented_at = created_at
+WHERE id > #{position}
+AND last_consented_at IS NULL
+AND deleted_at IS NOT NULL
+LIMIT #{batch_size}
+    SQL
+
```

Comment:
so the identities table is a hot table (accessed, written to often).

I think that we should do a big batch of little batches, ex 500,000 every time the job runs, but then only do 10,000 in a SQL query, so like:

```
(batch_size / slice_size).times.each do |slice_number|
  start_id = position + (slice_number * slice_size)
  limt = slice_size

  # ... query
end
```